libandroid-navigation/src/test/java/com/mapbox/services/android/navigation/v5/routeprogress/RouteStepProgressTest.java

```diff
@@ -29,83 +28,60 @@
 
   // Fixtures
```

Comment:
NIT unnecessary comment
What about adding the suffix `_FIXTURE` to the fixture constants?